content/blog/2020/05/2020-05-06-docker-agent-image-renaming.adoc

```diff
@@ -0,0 +1,84 @@
+* link:https://hub.docker.com/repository/docker/jenkins/agent[jenkins/agent] is a new name of the old link:https://hub.docker.com/repository/docker/jenkins/ssh-slave[jenkins/ssh-slave] image,
+  starting from link:https://github.com/jenkinsci/docker-ssh-agent/releases/tag/2.0.0[2.0.0]
+
+See the upgrade guidelines below.
+
+== Why?
+
+The "slave" term is widely considered inappropriate in open source communities.
+It has been officially deprecated in Jenkins 2.0 in 2016, but there are remaining usages in some Jenkins components.
+The jira:JENKINS-42816[Slave to Agent renaming leftovers] EPIC tracks cleanup of such usages.
+Official Docker agent images were a glaring case, it was not easy to fix that with the previous versions of the image release Pipelines on DockerHub.
+It great to have the image naming issue finally fixed by this update.
+
+Another notable change is replacing the _JNLP agent_ term by _inbound agent_.
+Historically "JNLP" has been used as a name of link:https://github.com/jenkinsci/remoting/blob/master/docs/protocols.md[Remoting protocols].
+JNLP stands for link:https://en.wikipedia.org/wiki/Java_Web_Start#Java_Network_Launching_Protocol_(JNLP)[Java Network Launch Protocol] which is a part of the Java Web Start.
+Jenkins supports Java Web Start mode for agents when running agents on Java 1.8,
+but our networking protocols are based on TCP and have nothing to do with Java Network Launch Protocol.
+So this name has been very confusing since the very beginning,
+and it became worse with the introduction of WebSocket support in Jenkins 2.217 (jep:222[]).
+Docker agent images support WebSockets, so we decided to change the image name to link:https://hub.docker.com/repository/docker/jenkins/inbound-agent[jenkins/inbound-agent] so that it prevents further confusion.
+
+== Upgrading and Compatibility Notes
+
+Good news, there are no breaking changes caused by this renaming.
+All images have been already modified to use the new terminology internally.
+If you use the recent versions of the previous images,
+you can just replace the old names by the new ones.
+These names may be referenced in your Dockerfiles, scripts and Jenkins configurations.
+
+We will keep updating the old images on DockerHub for at least 3 months.
+There will be no new configurations and platforms added to the old images,
+but all existing ones will remain available (Debian for Java 1.8 and 11, Alpine for Java 1.8, etc.).
+We may stop delivering updates to old images after 3 months (Aug 05, 2020),
+but previous versions will remain available to users.
+
+== What's next?
+
+We will continue renaming of the Docker images in Jenkins components which reference old image names.
+There is also a set of link:https://github.com/jenkinsci/jnlp-agents[convenience Docker images] which include build tools like Maven or Gradle which will be renamed later.
+The `jenkins/ssh-agent` image might be renamed again in the future due,
```

Comment:
Capitalize `jenkins`

Reply:
Since that is the name of an image, I don't think we should capitalize it.  I'd rather not rely on Docker to be case insensitive for organization names,

Reply:
aha! Good call and thank you